test/mavsdk_tests/mavsdk_test_runner.py

```diff
@@ -441,6 +430,25 @@ def start_runners(self,
                         self.verbose)
                     self.active_runners.append(gzclient_runner)
 
+                '''
+                WARNING: DO NOT PLACE PX4 RUNNER BEFORE GAZEBO RUNNER!
+                # We must start the PX4 instance at the end, as starting
+                it in the beginning, then connecting Gazebo server freaks
+                out the PX4 (it needs to have data coming in when started),
+                and can lead to EKF to freak out, or the instance itself
+                to die unexpectedly.
```

Comment:
```suggestion
                # We must start the PX4 instance at the end, as starting
                # it in the beginning, then connecting Gazebo server freaks
                # out the PX4 (it needs to have data coming in when started),
                # and can lead to EKF to freak out, or the instance itself
                # to die unexpectedly.
```

In my opinion this shouldn't be that shouty.